includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -0,0 +1,319 @@
+						],
+						'href' => $this->getHref( 'create-new-wiki' ),
+					]
+				]
+			]
+		];
+
+		if ( $this->lang === static::DEFAULT_LANG ) {
+			$data[ 'fandom_overview' ] = $this->getVerticalsSection();
+			$data[ 'wikis' ] = [
+				'header' => [
+					'type' => 'line-text',
+					'title' => [
+						'type' => 'translatable-text',
+						'key' => 'global-navigation-wikis-header',
+					],
+				],
+				'links' => [
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-wikis-explore'
+						],
+						'href' => $this->getHref( 'fan-communities' ),
+					],
+					$this->getCommunityCentralLink(),
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'translatable-text',
+							'key' => 'global-navigation-wikis-fandom-university'
+						],
+						'href' => $this->getHref( 'fandom-university' ),
+					]
+				]
+			];
+		} else {
+			$data[ 'wikis' ] = [
+				'links' => $this->getCommunityCentralLink()
+			];
+		}
+
+		if ( $wgUser->isLoggedIn() ) {
+			$data[ 'user' ] = $this->getLoggedInUserData( $wgUser );
+			$data[ 'notifications' ] = $this->getNotifications( $wgUser );
+		} else {
+			$data[ 'anon' ] = $this->getAnonUserData();
+		}
+
+		return $data;
+	}
+
+	private function getHref( $hrefKey ) {
+		return DesignSystemSharedLinks::getInstance()->getHref( $hrefKey, $this->lang );
+	}
+
+	private function getPageUrl( $pageTitle, $namespace, $query = '' ) {
+		return GlobalTitle::newFromText( $pageTitle, $namespace, $this->wikiId )->getFullURL( $query );
+	}
+
+	private function getSearchData() {
+		$isCorporatePage = $this->isCorporatePage();
+
+		$search = [
+			'type' => 'search',
+			'results' => [
+				'url' => $isCorporatePage
+					? $this->getCorporatePageSearchUrl()
+					: $this->getPageUrl( 'Search', NS_SPECIAL, [ 'fulltext' => 'Search' ] ),
+				'param-name' => 'query'
+			],
+			'placeholder-inactive' => [
+				'type' => 'translatable-text',
+				'key' => 'global-navigation-search-placeholder-inactive'
+			],
+			'placeholder-active' => [
+				'type' => 'translatable-text',
+				'key' => $isCorporatePage
+					? 'global-navigation-search-placeholder-wikis'
+					: 'global-navigation-search-placeholder-in-wiki'
+			]
+		];
+
+		if ( !$isCorporatePage ) {
+			$search['suggestions'] = [
+				'url' => WikiFactory::getHostById( $this->wikiId ) . '/index.php?action=ajax&rs=getLinkSuggest&format=json',
+				'param-name' => 'query'
+			];
+		}
+
+		return $search;
+	}
+
+	private function getAnonUserData() {
+		return [
+			'header' => [
+				'type' => 'line-image',
+				'image' => 'wds-icons-user',
+				'title' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-anon-my-account',
+				],
+				'subtitle' => [
+					'type' => 'translatable-text',
+					'key' => 'global-navigation-anon-my-account',
+				]
+			],
+			'links' => [
+				[
+					'type' => 'link-authentication',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "link-authentication" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)